### DISCUSSION TITLE
Test: Lost data problem

### DISCUSSION THREAD

**Discussion by Archie1978** [minio/minio#20568] [Q&A] [answered]
Hello,
     I am currently conducting tests with 3 nodes for greater simplicity.

I turned off node 1, then 5 minutes later I turned on node 1. Finally, 15 minutes later I turned off node 2. However, I can no longer access the data from the first 5 minutes. Is there a way to resynchronize node 2 with node 1 while all three are online?

kind, regards

**Comment by klauspost** [comment]
Scanning will heal missing shards, albeit slowly.  You can use `mc admin heal` to kick off healing manually.

**Reply by Archie1978** [reply]
hello klauspost,

 I repeated several tests with 4 nodes and I have a same problem.

My test: I create a bucket, then I turn off one node. Next, I add a file to this bucket and turn the node back on.
The folder (of the bucket) on MinIO is empty after 1 hour and after running the 'mc admin heal' command.

I'm doing this test to make sure I have 100% redundancy in MinIO when my cluster indicates 3/3 or 4/4 nodes.

Or at least, I would like to detect if an object is no longer redundant and is vulnerable to a node failure.

**Reply by Archie1978** [reply]
I forgot to mention that when I reset the node where data is missing, the data is indeed restored.

**Reply by Archie1978** [reply, marked as answer]
Sorry is it "mc admin heal ALIAS -r" , I m stupid :)
